.changeset/witty-wombats-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': minor
+'@backstage/theme': minor
```

Comment:
These should be `patch`, since we're pre-1.0 and it's not a breaking change 🙂

Reply:
Makes sense, I updated the changeset to be in line with that.